core/controllers/practice_sessions.py

```diff
@@ -47,8 +47,8 @@ class PracticeSessionsPage(
     }
 
     @acl_decorators.can_access_topic_viewer_page
-    def get(self, _: str) -> None:
-        """Handles GET requests."""
+    def get(self) -> None:
+        """Renders the practice session page."""
```

Comment:
Hey @deadex-ng, I think this is where the backend test is failing. Note that you have made changes to the `get` function where you have removed `, _: str`, restore it back and I think then the tests will pass.
Thanks

Reply:
@lkbhitesh07 Yes, you're right. The I have restored it and the backend tests have passed.
Thanks.